arch/arm/src/goldfish/goldfish_boot.c

```diff
@@ -41,6 +41,16 @@
 #  include <nuttx/sched_note.h>
 #endif
 
+#include <nuttx/syslog/syslog_rpmsg.h>
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+#ifdef CONFIG_SYSLOG_RPMSG
+static char g_syslog_rpmsg_buf[4096];
```

Comment:
Maybe the log buffer should be configurable, i.e. using CONFIG_SYSLOG_BUFSIZE

Reply:
@acassis The buffer is provided by chip, should not use a system config to config this buffer size?

Reply:
CONFIG_SYSLOG_BUFSIZE is a syslog line buffer length, not suitable here.

Reply:
@xiaoxiang781216 true! Thanks for warning my mistake